cocoviewer.py

```diff
@@ -168,7 +168,9 @@ def draw_masks(draw, objects, obj_categories, ignore, alpha):
             fill = tuple(list(c[-1]) + [alpha])
             # Polygonal masks work fine
             if isinstance(m, list):
-                draw.polygon(m[0], outline=fill, fill=fill)
```

Comment:
Oh, yeah, my bad! I think I did it for testing/debugging a couple of years ago, but I've been using the app for bboxes mainly, so I just didn't spot it. Thanks!